src/sentry/conf/server.py

```diff
@@ -1156,7 +1156,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     # Enable Sentry Functions
     "organizations:sentry-functions": False,
     # Enable experimental session replay backend APIs
-    "organizations:session-replay": False,
+    "organizations:session-replay": True,
```

Comment:
This will also enable replays in the next self-hosted release, and in single-tenant. Is that what you want?

Reply:
Oh, no i don't think that's what we want right now. I will close this PR because this is the only change.